base/shared/src/main/scala/scalaz/data/Is.scala

```diff
@@ -207,21 +208,11 @@ object Is {
   }
 
   /**
-    * It can be convenient to convert a [[=:=]] value into a `Leibniz` value.
-    * This is not strictly valid as while it is almost certainly true that
-    * `A =:= B` implies `A === B` it is not the case that you can create
-    * evidence of `A === B` except via a coercion. Use responsibly.
+    * It can be convenient to convert a [[=:=]] value into a [[===]] value.
     */
-  def fromPredef[A, B](ev: A =:= B): A === B = {
-    val _ = ev
-    unsafeForce[A, B]
-  }
+  def fromPredef[A, B](ev: A =:= B): A === B =
+    Axioms.predefEq(ev)
 
-  /**
-    * Unsafe coercion between types. `force` abuses `asInstanceOf` to
-    * explicitly coerce types. It is unsafe, but needed where Leibnizian
-    * equality isn't sufficient.
-    */
-  def unsafeForce[A, B]: A === B =
-    refl[Any].asInstanceOf[A === B]
```

Comment:
I think we should keep `unsafeForce`, unless we can replace its uses with something else (like `proposition` below). All current uses of `unsafeForce` have been replaced with `asInstanceOf`, which is not an improvement.